www/~/%username/emails/verify.html.spt

```diff
@@ -4,14 +4,17 @@ from datetime import timedelta
 
 from aspen import Response
 from aspen.utils import utcnow
-from gratipay.utils import emails, get_participant
+from gratipay.utils import emails, get_participant, b64decode_s
 
 [-----------------------------------------------------------------------------]
 
 participant = get_participant(state, restrict=False)
 banner = '~' + participant.username
 if participant == user.participant:
-    email = request.qs.get('email', '')
+    if 'email' in request.qs:
+        email = request.qs['email']
+    else:
+        email = b64decode_s(request.qs.get('email64', ''), default='')
```

Comment:
The purpose of doing this (which @Changaco suggested at https://github.com/gratipay/gratipay.com/pull/3965#issuecomment-197789029, but didn't provide a reason for) is to handle the race condition when deploying, yes? I.e., when we deploy this, we may have some users who requested an email verification link before we deployed, but aren't going to click on it until after we deploy. Since the old `email` key works fine for most folks, we don't want to fail their verification unnecessarily, yes?